plugins/infinispan52/src/main/java/org/radargun/service/HotRodOperations.java

```diff
@@ -37,7 +43,12 @@ public HotRodOperations(InfinispanHotrodService service) {
       }
    }
 
-   protected class HotRodCache<K, V> implements BasicOperations.Cache<K, V>, ConditionalOperations.Cache<K, V>, TemporalOperations.Cache<K, V> {
+   @Override
+   public <K, V> BulkOperations.Cache<K, V> getCache(String cacheName, boolean preferAsync) {
```

Comment:
You should honour the preferAsync flag (for putAll). Also, recent versions of RemoteCache have the getAll operation implemented natively.